src/contracts/gho/GhoToken.sol

```diff
@@ -107,8 +120,11 @@ contract GhoToken is ERC20, Ownable, IGhoToken {
   function setFacilitatorBucketCapacity(
     address facilitator,
     uint128 newCapacity
-  ) external onlyOwner {
+  ) external onlyBucketManagerOrOwner {
     require(bytes(_facilitators[facilitator].label).length > 0, 'FACILITATOR_DOES_NOT_EXIST');
+    if (_msgSender() == _bucketManager) {
```

Comment:
let's keep it unrestricted. This kind of restrictions (set to 0, manager per facilitator, etc) can be enforced with external helper contracts